lab.R

```diff
@@ -25,58 +25,78 @@ library(assertthat)
-#' 
+ri_df <- readxl::read_excel(file.path(data_folder, '7495_Robust-Intelligence.xlsx'), n_max = 25 )
 
 #' 2. *Suppose we're interested in looking at the amount of money NSF has awarded to different UC campuses for cognitive science-related work, by the year each project started.  What variables in the dataframe would be especially important for this research question?*
-#' 
-#' 
-#' 
-
-#' 3. *Before diving into dates, let's do a quick manual check of the money column, `AwardedAmountToDate`.  The name of this column can be misleading; for most types of awards, it represents the total amount of the award, not just the amount that has been distributed to date.  We can view the information for a specific award using a URL like this: <https://www.nsf.gov/awardsearch/showAward?AWD_ID=1813043> (this should match the top row of your dataframe).  Award IDs are in the `AwardNumber` column. For the first 8 awards, compare the `AwardedAmountToDate` values in the dataframe to the "Awarded Amount to Date" values shown in the award search system.  Based on these 8 awards, does `AwardedAmountToDate` seem to contain valid and correctly-formatted data?* 
-#' 
-#' 
-#' 
+#' AwardedAmountToDate, StartDate, Oganization 
 
 
+#' 3. *Before diving into dates, let's do a quick manual check of the money column, `AwardedAmountToDate`.  The name of this column can be misleading; for most types of awards, it represents the total amount of the award, not just the amount that has been distributed to date.  We can view the information for a specific award using a URL like this: <https://www.nsf.gov/awardsearch/showAward?AWD_ID=1813043> (this should match the top row of your dataframe).  Award IDs are in the `AwardNumber` column. For the first 8 awards, compare the `AwardedAmountToDate` values in the dataframe to the "Awarded Amount to Date" values shown in the award search system.  Based on these 8 awards, does `AwardedAmountToDate` seem to contain valid and correctly-formatted data?* 
+ri_df_first8 <- ri_df %>% head(8)
+# Subset the dataset so I can look the two columns more closely
+part <- ri_df_first8[c("AwardNumber", "AwardedAmountToDate")]
+#' No. For Award IDs being 1652633 and 1651909, the "Awarded Amount to Date" values in the award search system and the "AwardedAmountToDate" values are different. 
 
 #' # Problem 2 #
 #' 
 #' 1. *The column `StartDate` is supposed to contain the date each program began.  What class is this column?  How are the dates formatted?  For convenience as we write some functions to parse these dates, create a variable `dates` that contains the dates from `StartDate`.* 
-#' 
-#' 
-#' 
+class(ri_df$StartDate)
+#' The class of the "StartDate" column is "character". 
+#' The dates are formatted differently. Some dates refer to a certain day, and some some days are the days passed since a reference date. 
+ri_df_dates <- ri_df %>% 
```

Comment:
You're using different names for several variables, and Travis can't find them to check your work
```suggestion
dates <- ri_df %>% 
```